platforms/core-configuration/model-core/src/main/java/org/gradle/api/internal/provider/CollectionSupplier.java

```diff
@@ -18,10 +18,29 @@
 
 import java.util.Collection;
 
+/**
+ * A composable {@link ValueSupplier value supplier} that supplies collection values. Instances of
```

Comment:
❓ Is composable the right term here?  Isn't that an implementation detail of how the plus method can be implemented?  I don't think it's a concept that is relevant to the _interface_ itself.

Reply:
Would "combinable" sound better? That is what I am trying to emphasize here, that `CollectionSupplier`s  are value suppliers that can be combined (with `Collector`s). This is not about the implementation of `plus`, but its contract. I am wondering if I am misusing the term, as you can't combine `CollectionSupplier` with another `CollectionSupplier` (only with `Collector`s  - both are `ValueSupplier`s btw), so it is not an asymmetric operation.